MapboxNavigation/RouteMapViewController.swift

```diff
@@ -481,8 +481,8 @@ extension RouteMapViewController: NavigationComponent {
             mapView.showVoiceInstructionsOnMap(route: route)
         }
         
-        if mapView.routeLineTracksTraversal {
-            mapView.fade(route, fractionTraveled: progress.fractionTraveled)
+        if let parent = parent as? NavigationViewController, parent.routeLineTracksTraversal {
```

Comment:
To avoid making this assumption, we can add a `routeLineTracksTraversal` property to this class and turn `NavigationViewController.routeLineTracksTraversal` into a computed property that passes through to `RouteMapViewController.routeLineTracksTraversal`.